### PR TITLE
added login and signup pages

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**
Screenshot (19)
Screenshot (20)

### CODE REVIEW

**Comment by vivekgudipudi** [comment]
very clean!!!!!

File: auth/auth.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+.login-box{
+    display:flex;
+    height:80vh;
+    justify-content: center;
+    align-items:center;
+}
+.login-title{
+color:var(--dark-color);
+font-weight:600;
```
Comment (vivekgudipudi) [inline auth/auth.css]: if possible use utils for this

File: auth/auth.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}
+input:focus{
+    color: var(--primary-color);
+}
+
+.remember{
+    display:flex;
+    margin-bottom:var(--xs);
+    font-size:14px;
+    font-weight:400;
```
Comment (vivekgudipudi) [inline auth/auth.css]: you can add utils for this as well

**Review by vivekgudipudi — APPROVED** [review]

File: homepage.css [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
-}
+} */
 .ecom-navbar{
     display:flex;
     flex-wrap: wrap;
     justify-content: space-around;
     align-items: center;
     padding:var(--s);
+
+    /* position:sticky; */
```
Comment (samyakshah3008) [inline homepage.css:11]: You can remove this commented lines

**Review by samyakshah3008 — APPROVED** [review]
Code is clean, with good naming conventions and proper folder structure and clean UI 
Semantic html tags made it easier for review ✅ 

Good job!

File: auth/login.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                    <span class="user-data">Username</span>
+                    <input type="text" name="">
+                </div>
+                <div class="input-box">
+                    <span class="user-data">Password</span>
+                    <input type="password" name="">
+                </div>
+                <div class="remember">
+                    <label>
+                        <input type="checkbox" name="">
```
Comment (samyakshah3008) [inline auth/login.html:47]: In inputs, you can include placeholders as well for better User experience:)

File: auth/auth.css (hunk 1 above)
Comment (MiryalaAkshitha) [reply auth/auth.css]: sure done!

File: auth/login.html (hunk 4 above)
Comment (MiryalaAkshitha) [reply auth/login.html:47]: sure done!

**Pull request merged**